AsyncDisplayKit/ASCollectionView.h

```diff
@@ -79,6 +79,30 @@
 @property (nonatomic, assign) CGFloat leadingScreensForBatching;
 
 /**
+ *  Perform a batch of updates asynchronously, optionally disabling all animations in the batch. You can call it from background 
+ *  thread (it is recommendated) and the UI collection view will be updated asynchronously. The asyncDataSource must be updated 
+ *  to reflect the changes before this method is called.
+ *
+ *  @param animated   NO to disable animations for this batch
```

Comment:
Could you add some additional documentation description for why the animated property exists?  Wouldn't the individual update blocks specify any animation?  If so, could we delete that variant?